mitosheet/src/types.tsx

```diff
@@ -850,6 +851,14 @@ export interface UIState {
         // TODO: Move the other popups (loading, tour, fast forward) to use this infrastructure
         [PopupLocation.TopRight]: PopupInfo 
     }
+    dataRecon: { 
```

Comment:
Let's make a type for the data recon, and share the type with AITransformationResult. Peep the following fancy thing!

<img width="571" alt="Screen Shot 2023-05-23 at 1 18 26 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/35d76b69-9021-44ad-b32a-b051dca1424a">